### ISSUE TITLE
Add Figma Design for Marzban Frontend Using ShadCN Design System

### ISSUE THREAD

**Issue by x0sina** [Gozargah/Marzban#1484]
Currently, the Marzban frontend lacks a well-defined Figma design that adheres to a cohesive design system. This makes it challenging to implement consistent UI components across different screen sizes and themes.

I would like to request a Figma design for the new Marzban frontend based on the ShadCN design system. The design should include layouts for various screen sizes (responsive design) and support both light and dark themes. Leveraging the existing ShadCN design system in Figma will ensure consistency and streamline the development process.

An alternative would be to create a custom design system from scratch, but this approach would require significant time and effort. Using the ShadCN design system, which is already established and available in Figma, is a more efficient and reliable solution.

This feature will help developers build a modern and consistent UI for Marzban, ensuring usability across all devices. Screenshots or prototypes based on the ShadCN design system can be attached for reference if needed.

**Comment by gozarman** [comment]
Agree, here is a Figma file I was working on for a couple days. Its' basic configuration has been done and it's ready to implement the rest of the pages.

https://www.figma.com/design/8FqI7EjBbvD9a87Y95Zr2n/Marzban-UI---Shadcn?node-id=4704-4378&t=raqzNGneE0w0RApv-1
Anyone who wants to see the file, just submit a request, I will grant your access to view the file.

**Comment by ghost** [comment]
``

>
